src/main/java/com/revature/controllers/CurriculumController.java

```diff
@@ -95,42 +88,39 @@
 	public ResponseEntity<Curriculum> updateCurriculum(@Valid @RequestBody Curriculum updatedCurr) {
-		logger.info("[DEBUG] - in CurriculumController.updateCurriculum()");
 		Curriculum curr = service.updateCurriculum(updatedCurr);
 		return new ResponseEntity<>(curr, HttpStatus.OK);
 	}
 
-	/**
-	 * Gets all topics in the database associated with a given curriculum
-	 *
-	 * @param id
-	 *            int
-	 * @return ResponseEntity<List<Topic> A list of topic objects and a HTTP status
-	 *         code
-	 * @author Christian DeFaria 1806-Jun18-USF-Java Wezley Singleton
-	 */
-	@GetMapping(value = "/topics/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
-	public ResponseEntity<List<Topic>> getAllTopicsByCurriculumId(@PathVariable("id") int id) {
-		logger.info("[DEBUG] - in CurriculumController.getAllTopicsByCurriculumId()");
-		List<Topic> topics = service.getTopicsByCurriculumId(id);
-		return new ResponseEntity<>(topics, HttpStatus.OK);
-	}
-
-	/**
-	 * Gets all subtopics in the database associated with a given curriculum
-	 *
-	 * @param id
-	 *            int
-	 * @return ResponseEntity<List<Subtopic> A list of subtopic objects and a HTTP
-	 *         status code
-	 * @author Christian DeFaria 1806-Jun18-USF-Java Wezley Singleton
-	 */
-	@GetMapping(value = "/subtopics/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
-	public ResponseEntity<List<Subtopic>> getAllSubtopicsByCurriculumId(@PathVariable("id") int id) {
-		logger.info("[DEBUG] - in CurriculumController.getAllTopicsByCurriculumId()");
-		List<Subtopic> subtopics = service.getSubtopicsByCurriculumId(id);
-		return new ResponseEntity<>(subtopics, HttpStatus.OK);
-	}
+//	/**
+//	 * Gets all topics in the database associated with a given curriculum
+//	 *
+//	 * @param id
+//	 *            int
+//	 * @return ResponseEntity<List<Topic> A list of topic objects and a HTTP status
+//	 *         code
+//	 * @author Christian DeFaria 1806-Jun18-USF-Java Wezley Singleton
+//	 */
+//	@GetMapping(value = "/topics/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+//	public ResponseEntity<List<Topic>> getAllTopicsByCurriculumId(@PathVariable("id") int id) {
+//		List<Topic> topics = service.getTopicsByCurriculumId(id);
+//		return new ResponseEntity<>(topics, HttpStatus.OK);
+//	}
+//
+//	/**
+//	 * Gets all subtopics in the database associated with a given curriculum
+//	 *
+//	 * @param id
+//	 *            int
+//	 * @return ResponseEntity<List<Subtopic> A list of subtopic objects and a HTTP
+//	 *         status code
+//	 * @author Christian DeFaria 1806-Jun18-USF-Java Wezley Singleton
+//	 */
+//	@GetMapping(value = "/subtopics/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+//	public ResponseEntity<List<Subtopic>> getAllSubtopicsByCurriculumId(@PathVariable("id") int id) {
+//		List<Subtopic> subtopics = service.getSubtopicsByCurriculumId(id);
+//		return new ResponseEntity<>(subtopics, HttpStatus.OK);
+//	}
```

Comment:
Delete commented out code. If you need it later on you can store it in a git snippet